src/components/transactions/Warnings/FantomWarning.tsx

```diff
@@ -0,0 +1,22 @@
+import { Trans } from '@lingui/macro';
+import { Link, Typography } from '@mui/material';
+
+import { Warning } from '../../primitives/Warning';
+
+export const FantomWarning = () => {
```

Comment:
@0x4Graham well done and nice pr! Can we refactor the FantomWarning and harmony warning component into one? They share the same logic and code messages. So for example lets rename HarmonyWarning and FantomWarning into MarketWarning.tsx. We can pass the props. See below


``` 
import { Trans } from '@lingui/macro';
import { Link, Typography } from '@mui/material';

import { Warning } from '../../primitives/Warning';

interface MarketWarningProps {
  learnMore?: boolean; // Modify wording on link text,
  message: string;
  linkHref?: string;
}

export const MarketWarning = ({ learnMore, message, linkHref }: MarketWarningProps) => {
  return (
    <Warning severity="error">
      <Typography variant="caption">
        <Trans>
          {message}
          <Link href={linkHref} target="_blank">
            {learnMore ? 'Learn More' : 'Join the community discussion'}
          </Link>
        </Trans>
      </Typography>
    </Warning>
  );
};
```